app/controllers/api/v1/tags_controller.rb

```diff
@@ -40,8 +40,8 @@ def untag
         primary_instance = primary_collection_model.find(request_path_parts["primary_collection_id"])
 
         parsed_body.each do |i|
-          tag = Tag.find_by!(Tag.parse(i["tag"]))
-          primary_instance.tags.destroy(tag)
+          tag = Tag.find_by(Tag.parse(i["tag"]))
+          primary_instance.tags.destroy(tag) if tag.present?
```

Comment:
Is the `.present?` necessary?  `tag` will be either `nil` or a Tag instance, right?